spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -199,24 +197,74 @@
         end
 
         it 'results in 404 error' do
-          get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
+          get :tailoring_file, params: extra_params.merge(parents: [:policy], format: format)
 
           expect(response).to have_http_status :not_found
         end
       end
     end
 
-    context 'authorized via cert_auth' do
-      before do
-        allow(controller).to receive(:rbac_allowed?).and_call_original
-        allow(controller).to receive(:any_inventory_hosts?).and_return(true)
+    context 'XCCDF tailoring file' do
+      let(:format) { :xml }
+      let(:content_type) { 'application/xml' }
```

Comment:
There is a clear relationship between these two, so one can be eliminated.

Reply:
In case of XML, it could be eliminated.

However, since we're using shared tests for XML and JSON (and the relationship is not so straightforward there), I chose to differenciate it here as well:

```
let(:format) { :json }
let(:content_type) { 'application/vnd.api+json' }
```

Please, propose a change if you know of a better solution.

Reply:
```
Mime[format].to_s
```